koekalenteri-backend/src/utils/response.ts

```diff
@@ -0,0 +1,7 @@
+import { APIGatewayProxyResult } from "aws-lambda";
+
+export const response = (statusCode: number, body: any): APIGatewayProxyResult => ({
+    statusCode: statusCode,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/koekalenteri/koekalenteri/pullRequest?prid=7376473)
```suggestion
    statusCode,
```